client/src/components/game/star/StarDetail.vue

```diff
@@ -447,6 +467,24 @@ export default {
       }
 
       return GameHelper.getStarById(this.$store.state.game, this.star.wormHoleToStarId)
+    },
+    naturalResources: function() {
+      if  (!splitResources) {
+        return Math.floor(this.star.naturalResources.economy);
+      }
+      let economy = Math.floor(this.star.naturalResources.economy);
+      let industry = Math.floor(this.star.naturalResources.industry);
+      let science = Math.floor(this.star.naturalResources.science);
+      return economy + ' / ' + industry + ' / ' + science;
+    },
+    terraformedResources: function() {
```

Comment:
Can we get this from the API too? (floored)

Reply:
Resolved in the client

Reply:
Resolved the server